src/server/pkg/storage/chunk/chunk_test.go

```diff
@@ -9,12 +9,16 @@ import (
 	"github.com/pachyderm/pachyderm/src/client/pkg/require"
 )
 
+const (
+	averageBits = 23
```

Comment:
Given that it's now configurable, would it make sense to test with more than one value for averageBits?